src/components/nav.css

```diff
@@ -3,63 +3,54 @@
 nav {
   padding: 0 var(--base-content-padding);
   z-index: 200;
-}
-nav a {
-  padding: 12px 0 8px;
-  border-bottom: 3px solid transparent;
-  display: block;
-  font-size: 16px;
-  text-decoration: none;
-  font-weight: 600;   
-  text-decoration: none;
-  text-transform: uppercase;
-  color: #777;
+  background-color: #fff;
+  border-top: 1px solid #efefef;
 }
 .navbar {
   display: flex;
-  justify-content: flex-start;
   flex: 1 0 100%;
+  flex-direction: row;
+  justify-content: flex-start;
   list-style: none;
 }
-
-.navbar-item {
+nav ul li {
   position: relative;
   margin: 0;
   padding: 0 2rem 0 0;
 }
-
-/* header-specific nav styles */
-header nav {
-  border-top: 1px solid #efefef;
-  background-color: #fff;
-}
-header nav a {
-    color: #777;
-    font-weight: 600;
-    text-decoration: none;
-    text-transform: uppercase;
-}
-header nav a:hover, header nav a.active {
-  border-bottom-color: rgb(183,191,16);
+nav a {
+  padding: 12px 0 8px;
+  border-bottom: 3px solid transparent;
+  display: block;
+  font-size: 18px;
+  font-weight: 600;   
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #555;
```

Comment:
Setting color in this one place seems like a way to lose track of it. Make a variable for a secondary color if we want one?

Reply:
Sure. I'm fine with that. Maybe there's a way to set a gray and then have the ability to do darker or lighter versions of the same variable? I think I've seen a way to do that in SCSS or SASS, but don't know that I want to apply that here. So maybe just a couple additional color variables?

Reply:
I'm fine with adding some more color variables.

Reply:
Okay. I will add these variables. I'm also going through the rest of the CSS and JS files and adding these variables where applicable.